bandit/reporters/progress_base.h

```diff
@@ -92,6 +100,18 @@ namespace bandit {
       }
 
     protected:
+      void update_test_duration() {
+        testcase_duration_ = std::chrono::high_resolution_clock::now() - testcase_start_time_point_;
+        testsuite_runtime_ += std::chrono::duration_cast<std::chrono::nanoseconds>(testcase_duration_);
+      }
+
+      std::string print_time(double t) {
+        std::stringstream ss;
+        ss.precision(3);
+        ss << std::fixed << t;
```

Comment:
Okay it seems you did not like the rounding approach I had in mind. Your solution always shows 3 digits. If that is what you want (I don't care), I'm totally fine with it.

Reply:
This is not that I didn't like it, this is that the test were failing because a duration of zero was printing `0` and `0.` was expected.

I could also have updated the test but having a fixed length output for the duration might be confortable what do you think?

Reply:
As I said, I don't care. Just keep it this way. :)